style/header.scss

```diff
@@ -0,0 +1,11 @@
+.main-header {
+  height: 50px;
+  padding: 15px;
+  background-color: $bgwhite;
```

Comment:
There is no need here to set white background explicitly. Background color is transparent by default and your document is white